includes/settings/settings-ppec.php

```diff
@@ -263,6 +263,14 @@
 			'large'  => __( 'Large', 'woocommerce-gateway-paypal-express-checkout' ),
 		),
 	),
+	'cart_checkout' => array(
```

Comment:
can we make the name derives the boolean state from its type and possible values. Something like `cart_checkout_enabled`.